Test_Task/main.rb

```diff
@@ -0,0 +1,96 @@
+require 'watir'
+require 'nokogiri'
+require 'json'
+require 'date'
+require_relative './transactions'
+
+class WebBanking
+  attr_reader :browser, :page, :operation, :desc_operation
+  def initialize
+    @browser = Watir::Browser.new :chrome
+  end
+
+  def parse_accounts
+    sign_in
+    sleep(3)
+    accounts = browser.divs(class: %w(contract status-active))
+    info_list = accounts.map do |div|
```

Comment:
`info_list` variable sounds weird. variable should have useful name.
if you don't know if a name is good for a variable, use the following hack:
imagine you meet your variable after 300 lines of code, could you understand what is inside it?